src/k2/dto/Collection.h

```diff
@@ -191,9 +214,14 @@ class PartitionGetter {
     PartitionGetter(Collection&& collection);
     PartitionGetter() = default;
 
-    // Returns the partition and preferred endpointfor the given key.
+    // Returns the partition and preferred endpoint for the given key.
     // Hashes key if hashScheme is not range
-    PartitionWithEndpoint& getPartitionForKey(const Key& key, bool reverse=false, bool exclusiveKey=false);
+    PartitionWithEndpoint& getPartitionForKey(const Key& key, bool reverse = false, bool exclusiveKey = false);
```

Comment:
Partition is a network dto struct - we send and receive it. PartitionWithEndpoint is an in-memory struct which  holds pre-cached endpoint from the transport. It does not (and cannot) go over the wire since it is not serializable